app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less

```diff
@@ -1,8 +1,270 @@
+                }
+            }
+            .action.tocart.primary {
+                background-color: @color-maksym__main-black;
+                color: @color-maksym__white;
+                border: none;
+                border-radius: 0;
+                box-shadow: none;
+                font-size: 14px;
+                font-weight: 400;
+                padding: 15px 30px;
+                line-height: 1.1;
+                text-shadow: none;
+                text-transform: uppercase;
+                white-space: nowrap;
+                transition: background-color 0.3s ease;
+                &:hover,
+                &:focus {
+                    background-color: @color-maksym__main-gray;
+                }
+            }
+            .actions-secondary {
+                .action {
+                    width: 40px;
+                    height: 40px;
+                    line-height: 40px;
+                    text-align: center;
+                    font-size: 17px;
+
+                    margin-right: 10px;
+                    &::before {
+                        fill: @color-maksym__main-black;
+                        width: 17px;
+                        height: 17px;
+                    }
+                }
+            }
+        }
+    }
+    /* Bottom toolbar */
+    .toolbar.toolbar-products {
+        padding: 0;
+        .pages {
+            margin-right: 10px;
+            .item {
+                .page,
+                .action {
+                    display: inline-block;
+                    width: 36px;
+                    height: 36px;
+                    text-align: center;
+                    line-height: 36px;
+                    background-color: #eee;
+                    border: 0;
+                    color: #222;
+                    font-weight: 400;
+                    margin: 0;
+                    padding: 0;
+                    font-size: 14px;
+                    transition: background-color 0.4s ease 0s,
+                        color 0.4s ease 0s;
+                    &::before {
+                        color: #222;
+                        transition: color 0.4s ease 0s;
+                    }
+                    &:hover,
+                    &:focus {
+                        background-color: #222;
+                        color: #eee;
+                        &::before {
+                            color: #eee;
+                        }
+                    }
+                }
+                &:not(:last-child) {
+                    margin-right: 10px;
+                }
+                &.current {
+                    .page {
+                        background-color: #222;
+                        color: #eee;
+                    }
+                }
+            }
+        }
+        .limiter-options {
+            height: 36px;
+            margin: 0;
+            font-size: 13px;
+            color: #888888;
+            border-color: #eee;
+        }
     }
 }
```

Comment:
Please check this file and add variables and libs where appropriate

Reply:
Done